### ISSUE TITLE
Search Field for Phone - fullscreen dialog

### ISSUE THREAD

**Issue by InnaAtanasova** [SAP/fundamental-ngx#8607]
Clicking any search field in mobile mode should open a new **full screen dialog** in which items can be selected from a list of suggestions.
After opening the dialog, the initial focus should be set on the dialog and not in the input.
Input cursor should appear only after tapping in the input, and at that point the mobile keyboard should be displayed. Suggestions appear before typing.

**This is for any input that also displays a list of results**
- [ ] Combobox (core and platform)
- [ ] Multi input (core and platform)
- [ ] Multi combobox (platform)
- [ ] Search field (platform)
- [ ] Inputs in the shellbar

may be missing some

**Comment by mikerodonnell89** [comment]
Closing in favor of #8634
